templates/partial/_footer.html

```diff
@@ -0,0 +1,24 @@
+<footer class="p-strip">
```

Comment:
I would suggest to make the footer grey, similar to https://dqlite.io/ otherwise it looks stange, especially on the /docs page.
Also please remove the `is-bordered` class from index file, [line 156](https://github.com/canonical-web-and-design/microstack.run/pull/55/files#diff-fb5aa1cd1261d08d02db6f7dc314d9abL147)
```suggestion
<footer class="p-strip--light">
```